build.proj

```diff
@@ -63,9 +68,9 @@
     <MSBuild Projects="@(_TestProjects)" Properties="$(ProjectProperties)" />
   </Target>
 
-  <Target Name="Pack">
+  <Target Name="Pack" Condition="'$(BuildPackages)' == 'true'">
```

Comment:
Is this really needed? This target won't be hooked into the target execution if BuildPackages is set to false:
https://github.com/dotnet/corefx/pull/34382/files#diff-3ad318a39efd856540e8cf813f20ec09R20